src/components/Community/Section/styles.module.css

```diff
@@ -61,7 +61,7 @@
       margin-left: 10px;
       border-left: 7px solid transparent;
       border-right: 7px solid transparent;
-      border-top: 12px solid currentColor;
+      border-top: 12px solid currentcolor;
```

Comment:
This was a stylelint thing and it seems to [be supported in the CSS colors spec v4](https://www.w3.org/TR/css-color-4/#currentcolor-color), but I'm not 100% sure this doesn't break things in older browsers.